webpack.config.babel.js

```diff
@@ -181,7 +181,7 @@ const basePlugins = (bundle_name) => ([
     // ref: https://en.wikipedia.org/wiki/Base32
     // NOTE: please leave the BASE_32_VERSION be! It is updated automatically by
     // the release script 🤖
-    'BASE_32_VERSION': 'BI',
+    'BASE_32_VERSION': 'BK',
```

Comment:
How do we make sure we don't override an existing base32? 🤔

Reply:
I wasn't sure about this part of the release process tbh. 
This LTS release had to be done manually, as the branch is off `5.11.1` not `development` as usual. But we also need to increment the base32 version, but it shouldn't be the next increment as `5.12.0` (which did use the release script) now has it at `BJ` so even if we had used release script, the increment would have been incorrect & I think we'd end up overriding what's currently in the S3 bucket for base32 version `BJ`.